ui/app/pages/send/send.component.js

```diff
@@ -161,6 +162,17 @@ export default class SendTransactionScreen extends PersistentForm {
         this.updateGas()
       }
     }
+
+    const prevTokenAddress = prevSelectedToken && prevSelectedToken.address
+    const selectedTokenAddress = selectedToken && selectedToken.address
+
+    if (selectedTokenAddress && prevTokenAddress !== selectedTokenAddress) {
+      updateSendTokenBalance({
```

Comment:
Can we use `this.updateSendToken()` here?

Reply:
Nice catch! Updated